README.md

```diff
@@ -252,9 +252,31 @@ Code completion is available for the following elements:
 
 ## Customization
 
-It is possible to customize the behaviour of the `erlang_ls` server by
-adding a `erlang_ls.config` file to the root of your projects.
+It is possible to customize the behaviour of the `erlang_ls` server via a
+configuration file.
 
+### Configuration file location
+
+`erlang_ls` will use the first file found in the following locations:
+* path passed by the client during initialisation
+* <project root>/erlang_ls.config
+* <app config path>/erlang_ls/erlang_ls.config
+
+Generally speaking, <app config path> should be:
+
+| Operating System | Configuration Directory                             |
+|------------------|-----------------------------------------------------|
+| Linux            | /home/USER/.config/erlang\_ls                       |
+| OS X             | /Users/USER/Library/Application\ Support/erlang\_ls |
+| Windows          | c:/Users/USER/AppData/Local/erlang\_ls              |
```

Comment:
The `<app config path>` in the table already includes the `erlang_ls` directory, I presume it will not be repeated?